src/fuzzfetch/fetch.py

```diff
@@ -1106,8 +1100,8 @@ def _write_fuzzmanagerconf(self, path):
         output = configparser.RawConfigParser()
         output.add_section("Main")
         output.set("Main", "platform", self.moz_info["processor"].replace("_", "-"))
-        output.set("Main", "product", "mozilla-" + self._branch)
-        output.set("Main", "product_version", "%.8s-%.12s" % (self.id, self.changeset))
+        output.set("Main", "product", f"mozilla-{self._branch}")
+        output.set("Main", "product_version", f"{self.id:>.8}-{self.changeset:>.12}")
```

Comment:
the `<` and `>` have no effect, since we only specify a maximum field width, not a minimum or exact width.